body-tracking-samples/simple_3d_viewer/main.cpp

```diff
@@ -106,20 +107,107 @@ bool ParseInputSettingsFromArg(int argc, char** argv, InputSettings& inputSettin
         else if (inputArg == std::string("OFFLINE"))
         {
             inputSettings.Offline = true;
+            if (i < argc - 1) {
+                // Take the next argument after OFFLINE as file name
+                inputSettings.FileName = argv[i + 1];
+                return true;
```

Comment:
```suggestion
                i++;
```

Reply:
This way the file doesn't have the requirement to be on the end and the loop can keep processing.